packages/client/hmi-client/src/workflow/operator/tera-operator-inputs.vue

```diff
@@ -15,7 +15,7 @@
 			</div>
 			<span>
 				<label>
-					{{ input.label ?? input.type ?? '' }}
+					{{ input.label ?? input.defaultLabel ?? input.type ?? '' }}
```

Comment:
This kind of syntax is hard to read, maybe use a computed value or something.

Reply:
Isn't a computed just hiding away the same syntax?

```JS
<label>{{ label }}</label>
...
const label = computed(() => input.label ?? input.defaultLabel ?? input.type ?? '');
```

Reply:
I was complaining about null-coalesce chain being harder to read, but this comment is made rather redundant by my big blurb about leveraging typing systems.